tangelo/linq/simulator.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Simulator class, wrapping around the various simulators and abstracting their
+"""Simulator function, returns the various simulators that abstract their
```

Comment:
Let's adjust the docstring here. Make Simulator a class again, its job is now to "return an instance of the class that supports the target backend"